client/components/scoresheet/metadata_inputs/metadata_inputs_controller.js

```diff
@@ -98,13 +98,14 @@ class MetadataInputsController {
           return this.loadMatchOptions().then(() => this.autoselectMatch())
         }
       })
+      .catch(error => this.logger.error(error))
       .then(() => { this.autoselecting = false })
   }
 
   autoselectTeam () {
+    if (!this.refIdentity.table) return Promise.resolve()
```

Comment:
Not sure why the eslint isn't complaining on that one but give each if-statement it's own block, even if it's include 1-line